sentinel-core/src/main/java/com/alibaba/csp/sentinel/node/StatisticNode.java

```diff
@@ -113,7 +113,7 @@
     private long lastFetchTime = -1;
 
     @Override
-    public Map<Long, MetricNode> metrics() {
+    public synchronized Map<Long, MetricNode> metrics() {
```

Comment:
@cdfive because  the two timer have different bussiness scenes, so I am not put them in one timer and add the synchronized to ensure the thread-safe. could you please help to review this when you you are free? I ran the local test and can find the data in jvm tool.

Reply:
I run the `FlowQpsDemo`, and start jconsole, still can't find `Sentinel` node in jconsole UI. Then I make a breakpoint at `com.alibaba.csp.sentinel.node.metric.MetricTimerListener#aggregate`, and `Sentinel` node appeared, but the the PassQps,BlockQps are always 0.

I wonder that once the `StatisticNode#metrics`have been invoked, the data will be changed.
could you please try the demo and give some help? @sczyh30 @brothelul

Reply:
I will try the demo and re-produce the problem you met again, thank you so much!